include/deal.II/multigrid/multigrid.h

```diff
@@ -99,9 +99,8 @@ class Multigrid : public Subscriptor
             Cycle                              cycle = v_cycle);
 
   /**
-   * Experimental constructor for cases in which no DoFHandler is available.
-   *
-   * @warning Not intended for general use.
+   * Constructor. Same as above but you can determine the multigrid
+   * levels to use yourself.
```

Comment:
on another hand, when doing GMG in deal.II you are bound to have a dof handler object around and mesh and alike. So do we really win anything with this constructor?